### ISSUE TITLE
Publish new version to npm

### ISSUE THREAD

**Issue by brunolemos** [expo/react-native-action-sheet#66]
The version on npm (0.4.0) is crashing due to `React.PropTypes.any` code on `ActionSheetProvider.js` file.

This is weird because it was published 2 weeks ago and the fix in on master long before that.

But this is the content I'm getting on node_modules for the latest version:

```js
import React from 'react';
import ActionSheet from './ActionSheet';

export default class ActionSheetProvider extends React.Component {
  static propTypes = {
    children: React.PropTypes.any,
  };

  static childContextTypes = {
    showActionSheetWithOptions: React.PropTypes.func,
  };

  getChildContext() {
    return {
      showActionSheetWithOptions: (...args) =>
        this._actionSheetRef.showActionSheetWithOptions(...args),
    };
  }

  render() {
    return (
      <ActionSheet ref={component => (this._actionSheetRef = component)}>
        {React.Children.only(this.props.children)}
      </ActionSheet>
    );
  }
}
```

![simulator screen shot - iphone 8 - 2017-12-14 at 18 22 31](https://user-images.githubusercontent.com/619186/34012646-c4adc192-e0fb-11e7-88a2-edbb5b8e8802.png)

**Comment by kumarryogeshh** [comment]
same here.

**Comment by jamesalways** [comment]
Same! Thanks!

**Comment by jamesalways** [comment]
Same error in ActionSheet.android.js

**Comment by brunolemos** [comment]
As a workaround point it to master that it works:
`"react-native-action-sheet": "github:expo/react-native-action-sheet"`

**Comment by jamesalways** [comment]
Not work in Android! See https://github.com/expo/react-native-action-sheet/issues/68

**Comment by eightyfive** [comment]
When will new version be published?? Thx.

**Comment by peterpme** [comment]
Please use 1.0.2 moving forward, thanks!

**Comment by peterpme** [comment]
:wave: 
@eightyfive 
@jamesalways 
@brunolemos 
@kumarryogeshh 

This should be fixed in 1.0.2:
https://www.npmjs.com/package/@expo/react-native-action-sheet
